examples/alarm/wrangler.toml

```diff
@@ -0,0 +1,21 @@
+name = "alarm-example"
+workers_dev = true
+compatibility_date = "2022-09-12" # required
+compatibility_flags = ["streams_enable_constructors"]
+main = "./build/worker/shim.mjs"
```

Comment:
I'm not quite sure if there are things that can be removed to make a better minimal Wrangler.toml. There are a few cleaning items to resolve here though.